src/views/AddPriceMultiple.vue

```diff
@@ -50,18 +50,37 @@
                 :loading="createProofLoading">
               </v-file-input>
               <p v-if="proofFormFilled && !createProofLoading" class="text-green mt-2 mb-2">
-                  <i v-if="!proofSelectedMessage">{{ $t('AddPriceSingle.PriceDetails.ProofUploaded') }}</i>
-                  <i v-if="proofSelectedMessage">{{ $t('AddPriceSingle.PriceDetails.ProofSelected') }}</i>
+                  <i v-if="!proofisSelected">{{ $t('AddPriceSingle.PriceDetails.ProofUploaded') }}</i>
+                  <i v-if="proofisSelected">{{ $t('AddPriceSingle.PriceDetails.ProofSelected') }}</i>
                 </p>
               <p v-if="!proofFormFilled && !createProofLoading" class="text-red mt-2 mb-2">
                 <i>{{ $t('AddPriceSingle.PriceDetails.UploadProof') }}</i>
               </p>
               <p v-if="proofType === 'RECEIPT'" class="text-caption text-warning">
                 <i>{{ $t('AddPriceMultiple.ProofDetails.ReceiptWarning') }}</i>
+                <i>{{ $t('AddPriceMultiple.ProofDetails.PrivateWarning') }}</i>
               </p>
             </v-col>
-            <v-col v-if="proofFormFilled">
+            <v-col  cols="12" sm="12" md="4" v-if="proofFormFilled">
               <v-img :src="proofImagePreview" style="max-height:200px"></v-img>
+              <div class="proofIsPublic-switch-container" v-if="proofFormFilled && (proofType === 'RECEIPT')">
```

Comment:
I would have moved this to a row below.
- warning text
- then switch (with label next to it)

Reply:
example : (juste avec des row et des col, plus besoin de css custom)

![image](https://github.com/openfoodfacts/open-prices-frontend/assets/7147385/9d482a15-f4e0-4d5b-9539-1befa1edc42d)